demo-project/.telemetry

```diff
@@ -1 +1 @@
-consent: false
+consent: true
```

Comment:
I hope this is for testing and will be reverted ?

Reply:
we have a dev telemetry dashboard actually. which is where i test. but i will revert this.